### DISCUSSION TITLE
What happened to "Don't forget for 1.0"?

### DISCUSSION THREAD

**Discussion by EliahKagan** [GitoxideLabs/gitoxide#1715] [Q&A] [answered]
[In the past](https://github.com/GitoxideLabs/gitoxide/discussions/1390), there were two GitHub "projects":

- Collaboration Board
- Don't forget for 1.0

I believe that "Collaboration Board" turned into the current "Kanban" project, but I'm not sure what happened to "Don't forget for 1.0." Was that not needed, and intentionally deleted? Or did it get lost as the repository was moved into the GitoxideLabs org?

**Comment by Byron** [comment, marked as answer]
Thanks for bringing this up - actually I thought nobody would notice, otherwise I probably would have done it with more care.

The collaboration board was essentially orphaned and it wasn't touched in a while, so it was removed.

The removal of the "Don't forget for 1.0" board was a bit of a mistake as I think it contained cards that weren't backed by an issue. However, I am quite optimistic that I won't forget anything major - the plan is to put issues into milestones, there already is one for 1.0.
